cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,57 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticleEditPageObject from '../support/pages/articleEdit.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
 
-describe('Article', () => {
-  before(() => {
+const ArticleEditPage = new ArticleEditPageObject();
+const homePage = new HomePageObject();
 
-  });
+describe('Article', () => {
+  let article;
+  let user;
 
   beforeEach(() => {
     cy.task('db:clear');
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   it('should be created using New Article form', () => {
-
+    cy.register(user.email, user.username, user.password);
+    cy.login(user.email, user.password);
```

Comment:
Move it into beforeEach from every test in this file for avoiding code duplication.